Dockerfile

```diff
@@ -87,4 +87,23 @@ RUN : \
     && rm -rf /opt/go/doc /opt/go/test \
     && rm go.tgz
 
+ARG RUST=1.48.0
+ARG RUSTUP_SHA256=ee7ade44063c96c6a37012cc599cb560dce95205c86d17b247c726d2285b230c
+ARG RUSTUP_VERSION=1.23.0
+ENV \
+    RUSTUP_HOME=/usr/local/rustup \
+    CARGO_HOME=/usr/local/cargo \
+    PATH=/usr/local/cargo/bin:$PATH
+RUN : \
+    && rustArch='x86_64-unknown-linux-gnu' \
+    && curl --silent --location --output rustup-init "https://static.rust-lang.org/rustup/archive/${RUSTUP_VERSION}/${rustArch}/rustup-init" \
+    && echo "${RUSTUP_SHA256} rustup-init" | sha256sum --check \
+    && chmod +x rustup-init \
+    && ./rustup-init -y --profile minimal --no-modify-path --default-toolchain $RUST --default-host ${rustArch} \
+    && chmod -R a+w $RUSTUP_HOME $CARGO_HOME \
```

Comment:
these should be quoted (both this line and the one above it) even though the variables currently don't matter

also `${rustArch}` (1) should be quoted and (2) the braces aren't doing anything